ratelimit/redis.go

```diff
@@ -380,11 +383,10 @@ func (*clusterLimitRedis) Resize(string, int) {}
 // ratelimits being not strongly consistent across calls to Allow()
 // and RetryAfter() (or AllowContext and RetryAfterContext accordingly).
 //
-// Performance considerations: It uses Oldest() to get the data from
-// Redis.
-//
 // If a context is provided, it uses it for creating an OpenTracing span.
-func (c *clusterLimitRedis) RetryAfterContext(ctx context.Context, s string) int {
+func (c *clusterLimitRedis) RetryAfterContext(ctx context.Context, clearText string) int {
+	s := getHashedKey(clearText)
```

Comment:
this we don't need here, because we only send out the clearText when calling c.oldest() through c.deltaFrom(), and c.oldest() does the hashing on its own.

Reply:
thanks!